Model/Push.php

```diff
@@ -187,7 +189,8 @@ public function __construct(
         OrderStatusFactory $orderStatusFactory,
         PaymentGroupTransaction $groupTransaction,
         \Magento\Framework\ObjectManagerInterface $objectManager,
-        ResourceConnection $resourceConnection
+        ResourceConnection $resourceConnection,
+        \Magento\Framework\Filesystem\DirectoryList $dirList
```

Comment:
Can we use the alias here to follow the other arguments syntax?

Reply:
usually we use aliases, but in general they are useful if we will call this classes later 
but in this specific case we don't use it and use only instance of it
so no real benefit of your proposal
as well as Magento's core (if you will check this) doesn't always follow such things,